wit/deps/spin@unversioned/mqtt-types.wit

```diff
@@ -0,0 +1,17 @@
+interface mqtt-types {
```

Comment:
I'm not sure adding the `mqtt` interface to `spin@unversioned` is necessary here. The `spin@unversioned` package is here to maintain backwards compatibility with Spin V1 components. New features should be added to root level `wit` package as you have done below.

Reply:
Thanks, will update.